include/vcpkg/base/chrono.h

```diff
@@ -40,11 +40,11 @@ namespace vcpkg
     {
         static ElapsedTimer create_started();
 
-        constexpr ElapsedTimer() noexcept : m_start_tick() { }
+        constexpr ElapsedTimer() noexcept = default;
```

Comment:
This change is potentially damaging, since it replaces value initialization with default initialization. (Zero init is being replaced with potentially-garbage-init)